packages/js/product-editor/src/products-app/utilites/storybook.tsx

```diff
@@ -0,0 +1,1662 @@
+/**
```

Comment:
This file mostly contains test data, could we keep it as such and move the two react components out at the bottom?
Maybe also rename this to `product-data-view-data.ts`?